src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -148,7 +148,8 @@ private EditAndContinueWorkspaceService GetEditAndContinueService(Workspace work
             var sessionId = await service.StartDebuggingSessionAsync(
                 solution,
                 _debuggerService,
-                captureMatchingDocuments: false,
+                captureMatchingDocuments: ImmutableArray<DocumentId>.Empty,
```

Comment:
This parameter name doesn't make sense any more. Perhaps just `documentIds`? or `matchingDocuments`?

Reply:
Yeah, name not ideal but couldn't figure out a better one. The meaning is "ids of documents to capture if their content matches the PDB". Just "documentIds" does not explain what kind of documents and "matchingDocuments" is misleading since these documents do not need to match - they need to be checked for match.

Reply:
LMK if you can think of a better name and I'll follow up with the rename.